src/test/java/calculator/StringCalculatorTest.java

```diff
@@ -0,0 +1,57 @@
+package calculator;
+
+import caculator.Operator;
+import caculator.StringCalculator;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.ValueSource;
+
+import static org.assertj.core.api.Assertions.assertThatIllegalArgumentException;
+import static org.assertj.core.api.AssertionsForClassTypes.assertThat;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+public class StringCalculatorTest {
+    @Test
+    public void 더하기(){
+        assertThat(Operator.calculate("+", 1,2)).isEqualTo(3);
```

Comment:
연산기호별 테스트는 OperatorTest로 분리하면 어떨까요?
공백 [컨벤션](https://google.github.io/styleguide/javaguide.html#s4.6-whitespace)도 확인해 주시면 좋을 것 같아요
```suggestion
        assertThat(Operator.calculate("+", 1, 2)).isEqualTo(3);
```

Reply:
넵! OperatorTest도 따로 추가하여 분리했습니다~